PoseEstimation-TFLiteSwift/PoseKeypointsDrawingView.swift

```diff
@@ -16,10 +16,11 @@ class PoseKeypointsDrawingView: UIView {
             self.setNeedsDisplay()
         }
     }
+    var threshold: CGFloat?
     
     override func draw(_ rect: CGRect) {
-        guard let keypoints = result?.keypoints,
-            let lines = result?.lines else { return }
+        guard let keypoints = result?.filterKeypoints(with: threshold),
+            let lines = result?.filterLines(with: threshold) else { return }
```

Comment:
keypoints & lines was filtered by result's method with threshold...

This logic isn't suitable for UI.

Why not have filtered keypoints & lines? 
is any special reason that poseKeypointsDrawingView filter keypoints & lines out in draw(_ rect: CGRect)?